src/panels/config/cloud/google-assistant/cloud-google-assistant.ts

```diff
@@ -128,32 +129,38 @@ class CloudGoogleAssistant extends LitElement {
                 .map((trait) => trait.substr(trait.lastIndexOf(".") + 1))
                 .join(", ")}
             </state-info>
-            <ha-formfield
-              .label=${this.hass!.localize(
-                "ui.panel.config.cloud.google.expose"
-              )}
-            >
-              <ha-switch
-                .entityId=${entity.entity_id}
-                .disabled=${!emptyFilter}
-                .checked=${isExposed}
-                @change=${this._exposeChanged}
+            <div>
+              <ha-formfield
+                .label=${this.hass!.localize(
+                  "ui.panel.config.cloud.google.expose"
+                )}
+                .dir="${computeRTLDirection(this.hass!)}"
               >
-              </ha-switch>
-            </ha-formfield>
+                <ha-switch
+                  .entityId=${entity.entity_id}
+                  .disabled=${!emptyFilter}
+                  .checked=${isExposed}
+                  @change=${this._exposeChanged}
+                >
+                </ha-switch>
+              </ha-formfield>
+            </div>
             ${entity.might_2fa
               ? html`
-                  <ha-formfield
-                    .label=${this.hass!.localize(
-                      "ui.panel.config.cloud.google.disable_2FA"
-                    )}
-                  >
-                    <ha-switch
-                      .entityId=${entity.entity_id}
-                      .checked=${Boolean(config.disable_2fa)}
-                      @change=${this._disable2FAChanged}
-                    ></ha-switch>
-                  </ha-formfield>
+                  <div>
+                    <ha-formfield
+                      .label=${this.hass!.localize(
+                        "ui.panel.config.cloud.google.disable_2FA"
+                      )}
+                      .dir="${computeRTLDirection(this.hass!)}"
```

Comment:
If we use it multiple times in one element, we can do this on top of render:
```ts
const dir = computeRTLDirection(this.hass!);
```
and use that
```suggestion
                      .dir=${dir}
```